Should link blas otherwise ld return error adding symbols: DSO missing from command line
--- orig/clients/benchmarks/CMakeLists.txt
+++ rocSOLVER-rocm-4.3.0/clients/benchmarks/CMakeLists.txt
@@ -17,6 +17,7 @@ target_include_directories( rocsolver-be
     )
 
 target_link_libraries( rocsolver-bench PRIVATE
+  blas
   cblas
   lapack
   Threads::Threads
--- orig/clients/gtest/CMakeLists.txt
+++ rocSOLVER-rocm-4.3.0/clients/gtest/CMakeLists.txt
@@ -87,6 +87,7 @@ target_include_directories( rocsolver-te
     )
 
 target_link_libraries( rocsolver-test PRIVATE
+  blas
   cblas
   lapack
   GTest::GTest
